src/imagery/i.eodag/i.eodag.py

```diff
@@ -42,6 +42,18 @@
 # % guisection: Print
 # %end
 
+# %flag
+# % key: b
+# % description: Use the borders of the AOI polygon and not the region of the AOI
+# % guisection: Filter
+# %end
+
+# %flag
+# % key: s
+# % description: Skip scenes that have already been downloaded after ingestiondate
```

Comment:
```suggestion
# % description: Skip scenes that have already been downloaded
```

Reply:
Shall we keep this since we are keeping the modification date condition?

Reply:
the description was not right either, even if we keep the logic... so, no, I'd still remove the "after ingestiondate" part